package.json

```
@@ -21,19 +21,17 @@
     "url": "http://projects.theforeman.org/projects/foreman_discovery/issues"
   },
   "peerDependencies": {
-    "@theforeman/vendor": "^6.0.0"
+    "@theforeman/vendor": ">= 6.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.7.0",
-    "@theforeman/builder": "^6.0.0",
-    "@theforeman/eslint-plugin-foreman": "^6.0.0",
-    "@theforeman/stories": "^7.0.0",
-    "@theforeman/test": "^8.0.0",
-    "@theforeman/vendor-dev": "^6.0.0",
-    "babel-eslint": "^10.0.3",
-    "eslint": "^6.7.2",
-    "prettier": "^1.19.1",
-    "stylelint": "^9.3.0",
-    "stylelint-config-standard": "^18.0.0"
+    "@babel/core": ">= 7.7.0",
+    "@theforeman/builder": ">= 6.0.0",
+    "@theforeman/eslint-plugin-foreman": ">= 6.0.0",
+    "@theforeman/stories": ">= 7.0.0",
+    "@theforeman/test": ">= 8.0.0",
+    "eslint": ">= 6.7.0",
```

```suggestion
    "eslint": "^6.7.0",
```

This should fix the linting error.
I would suggest changing all the dev deps to use `^` instead of `>=` so nothing will break when a 3rd party adds breaking changes.

---

I have amended the last commit, thank you for review.